File: src/swingjs/JSEvent.java
package swingjs;

import javajs.util.JSThread;
import jsjava.awt.event.InvocationEvent;

public class JSEvent extends InvocationEvent {
	
	JSEvent(JSThread t, Runnable r) {
		super(t, InvocationEvent.SWINGJS_INVOCATION_LOW, r, null, false);
	}

}

File: site/swingjs/j2s/swingjs/JSEvent.js
Clazz.declarePackage ("swingjs");
Clazz.load (["java.awt.event.InvocationEvent"], "swingjs.JSEvent", null, function () {
c$ = Clazz.declareType (swingjs, "JSEvent", java.awt.event.InvocationEvent);
Clazz.makeConstructor (c$, 
function (t, r) {
Clazz.superConstructor (this, swingjs.JSEvent, [t, 1201, r, null, false]);
}, "swingjs.JSThread,Runnable");
});
